oci/private/image_index.sh.tpl

```diff
@@ -35,7 +35,7 @@ function copy_blob() {
     local blob_image_relative_path="$3"
     local dest_path="${output_path}/${blob_image_relative_path}"
     mkdirp "$(dirname "${dest_path}")"
-    "${COREUTILS}" ln -f "${image_path}/${blob_image_relative_path}" "${dest_path}"
+    "${COREUTILS}" cat "${image_path}/${blob_image_relative_path}" > "${dest_path}"
```

Comment:
nit: why `cat` instead of `cp` ?

Reply:
They are write protected which prevents cp'ying.